frontend/src/static/js/components/webstatus-line-chart-panel.ts

```diff
@@ -115,6 +116,13 @@ export interface FetchFunctionConfig<T> {
  * panel identification, text display, and chart options.
  */
 export abstract class WebstatusLineChartPanel extends LitElement {
+  /**
+   * The list of supported browsers for the chart.
+   * @abstract
+   * @type {Array<BrowsersParameter>}
+   */
+  abstract browsers: Array<BrowsersParameter>;
```

Comment:
We should not couple the generic line chart panel to be browser specific. Instead of storing this as a property to pass it in getDisplayDataChartOptionsInput, allow each child chart panel to maintain this data. For example, if we want to start a chart about a list of features instead of a list of browsers, this detail would limit that.

Reply:
If you really wanted to keep this type of variable, you could also do this

```ts
export abstract class WebstatusLineChartPanel<S> extends LitElement {
  /**
   * The list of supported browsers for the chart.
   * @abstract
   * @type {Array<S>}
   */
  abstract series: Array<S>;
...
...

export abstract class WebstatusLineTabbedChartPanel<S> extends WebstatusLineChartPanel<S> {
...
}
```

Then concrete  classes could be like 

```ts
export class WebstatusStatsConcreteBrowserPanel extends WebstatusLineTabbedChartPanel<BrowserParameter>
```